Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -47,8 +47,8 @@ def default(context, obj):
                     type='github',
                     url=obj.url,
                 )
-                if obj._res[2:]:
-                    result['res'] = [compiled.pattern for compiled in obj._res[2:]]
+                if obj._res[4:]:
```

Comment:
I wonder if we could make it more dynamic, e.g. self._default_res = [n default regex], self._res = self._default_res + (res or []), the we we do `obj._res[len(self._default_res):]:`

Or create
```
@property

def _res(self):
   return self._default_res + self._extra_res

def __init__(...):
    self._default_res = [...]
    self._extra_res = res or []


if obj._extra_res:
   ...
``

Reply:
The `Bugzilla` object has a very similar problem, so I ended up with a slight variation on your first suggestion. The base template regexes were ugly in the constructor as is, so I stuck their templates in a list belonging to the class declaration, now we can just add to the list and we won't have to hard-code lengths anywhere.